tests/test_neighborhood.py

```diff
@@ -757,6 +757,7 @@ def test_all_same_throughputs(self):
         step_vector = n._get_step_vector()
         self.assertEqual(step_vector, Coordinate([0, 0, 0]))
 
+        n.force_slow_mode()
```

Comment:
do we lose fast-mode coverage with this? Should it just be a new test (or the same test called twice with a bool for slow/fast mode)?